### ISSUE TITLE
🐞 Bug: Tabwechsel nicht möglich

### ISSUE THREAD

**Issue by laske185** [public-ui/kolibri#5616]
> **Note**
> - [ ] Ich achte beim Erstellen des Issues darauf, dass ich **keine** fachlichen und unternehmensinternen Details preisgebe.

---

# Fehlermeldung

## Beschreibung des Fehlers
Es ist nicht möglich beim Tab-Control die Tabs zu wechseln, wenn der selektierte Tab mittels `selected` nicht angegeben wird. Das Control bleibt immer bei Tab 0, da die property `_selected` in den DOM geschrieben wird.

## Reproduktion

[Link zu StackBlitz](https://stackblitz.com/edit/vitejs-vite-qeps2m?file=package.json,src%2FApp.tsx)

```tsx
<KolTabs _tabs={[{ _label: 'Tabellen' }, { _label: 'Radio-Buttons' }]}>
    <ReproduktionsbeispielTabellen />
    <ReproduktionsbeispielRadioButtons />
  </KolTabs>
```

ergibt im DOM:

```html
<kol-tabs _selected="0" class="hydrated" style="">
    <article slot="tabpanel-slot-0">
    ...
    </article>
    <article slot="tabpanel-slot-1">
    ...
    </article>
</kol-tabs>
```

Schritte zum Reproduzieren des Verhaltens:
- Öffne Stackblitz
- Versuche zum 2. Tab zu wechseln

## Erwartetes Verhalten
Das 2. Tab kann geöffnet werden, ohne dass das selektierte Tab in die Komponente gegeben wird.

**Comment by laske185** [comment]
Es kommt beim Click zu einer Exception in der Zeile:
```js
this.controller.setFormAssociatedValue(this.state._value),
```
Uncaught TypeError: Cannot read properties of undefined (reading 'setFormAssociatedValue')

**Comment by sdvg** [comment]
Der Fehler tritt nur im Experimental Mode auf.
Mit [deaktiviertem Experimental Mode](https://stackblitz.com/edit/vitejs-vite-qvwn2v?file=index.html) funktioniert alles. 

Lösung: Diese Zeile in der index.html ggf. wie folgt anpassen:

```
<meta name="kolibri" content="dev-mode=true,experimental-mode=false" />
```

Der Experimental Mode ist nicht für Produktion gedacht und Probleme sind zu erwarten, ich würde das Ticket daher schließen.
